### ISSUE TITLE
The data returned by <store>.get() is writable

### ISSUE THREAD

**Issue by akowi-sknobloch** [nanostores/nanostores#143]
I've stumbled upon this error when testing out the new @nanostores/lit package, therefore i first thought the problem lies there.
I now believe the bug is actually in the normal nanostores.

When using `<store>.get()` i can manipulate the state by changing the returned object.

In my Example the store returns an Array that I want to sort and cap to a certain size.
But the `splice` method i use to cap the Array manipulates the Array contained in my atom.

I would expect the `.get()` method to just return a readable copy of the state, therefore i think this is a bug.  

Here is an example Repository showing this behaviour: https://github.com/akowi-sknobloch/nanostores-lit-clears-store-bug

**Comment by ai** [comment]
Copying the stay on every read will slow down state manager a lot.

Do you have any other idea how to solve this problem? What other state managers do?

**Comment by akowi-sknobloch** [comment]
Ok that makes sense.
I've checked the Documentation of Redux as that's the only other state manager I've used in the past. 
They have a Section in their Style Guide about not mutating the state: https://redux.js.org/style-guide/#do-not-mutate-state
Sounds to me that Redux has the same "Problem".

The current Docs already state to prevent using `.get()`.
My confusion stemmed from the `@nanostores/lit` examples using `.get()`.
Best solution would properly be documenting this behaviour and maybe update the examples in `@nanostores/lit` to prevent future misunderstandings.
